app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(
+      post: Post.find(params[:post_id]),
+      author: User.find(params[:author_id])
+    )
+
```

Comment:
[OPTIONAL] I think your app will be safer if it uses strong parameters. You can check this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) out